testing/php7-rrd/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Contributor: Pascal Hofmann <mail@pascalhofmann.de>
+# Contributor: Micka�l Remars <github@remars.com>
+# Maintainer: Pascal Hofmann <mail@pascalhofmann.de>
+pkgname=php7-rrd
```

Comment:
Name should be `php7-pecl-rrd`

Reply:
Renamed the package. The extension seems still compatible with recent php versions. I just had to install an additional font package to get complete graphs.